clean.bat

```diff
@@ -1,13 +1,13 @@
-@echo off
-for %%D in (VirtIO NetKVM viostor vioscsi Balloon vioserial viorng vioinput viofs pvpanic pciserial fwcfg packaging Q35 ivshmem fwcfg64 viosock viogpu) do (
-  pushd %%D
-  if exist cleanall.bat (
-    call cleanall.bat
-  ) else (
-    call clean.bat
-  )
-  popd
-)
-
-if exist buildfre_*.log del buildfre_*.log
-if exist buildchk_*.log del buildchk_*.log
+@echo off
+for %%D in (VirtIO NetKVM viostor vioscsi Balloon vioserial viorng vioinput viofs pvpanic pciserial fwcfg packaging Q35 ivshmem fwcfg64 viosock viogpu viomem) do (

```

Comment:
The change is only in this line, but all block appears as changed. Probably indentation change, if needed better add it as separate commit.